test/server/cards/02_SHD/units/TobiasBeckettITrustNoOne.spec.ts

```diff
@@ -0,0 +1,60 @@
+describe('Tobias Beckett, I Trust No One', function () {
+    integration(function (contextRef) {
+        describe('Tobias Beckett\'s ability', function () {
```

Comment:
Can we also add a test that playing him from Smuggle calculates his Smuggle aspects correctly?

Reply:
playing tobias or playing a non-unit card ?

Reply:
@VJubert I was thinking of smugglign Tobias but it would be good to test that his ability works with Smuggle too

Reply:
because smuggling with different aspect was covered on Smuggle.spec.ts

Reply:
For playing Tobias, because his smuggle aspect is different than normal. That test should probably go in `Smuggle.spec.ts`

Reply:
@AMMayberry1 I think it's just preference for where to cover things. It's generic but also part of the card. Like, for example would we test a card whose ability was just "when played: deal 1 damage"? We know dealing damage works, so why test it? I think we should probably test all aspects of a card in the card, even if the logic is tested elsewhere